lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -390,7 +390,7 @@ sub engine_workit {
             # PERL5OPT may have Devel::Cover options, we don't need and want
             # them in the spawned process as it does not belong to openQA code
             local $ENV{PERL5OPT} = "";
-            exec "perl", "$isotovideo", '-d';
+            exec $job_settings->{ISOTOVIDEO} ? split(' ', $job_settings->{ISOTOVIDEO}) : "perl", "$isotovideo", '-d';
```

Comment:
This simple splitting will not handle quoting/escaping correctly. Maybe it makes more sense to just pass the path as-is? And for readability wouldn't it make sense to swap out the `, "$isotovideo" ` part as well? Or is the code already doing that (braces would help)?

Reply:
yes, you are right, passing as-is seems to be a better choice. https://perldoc.perl.org/functions/exec is much smarter than my split.

I guess braces help. By the way, to test locally:

```bash
env MY_COMMAND="echo -e 'my custom\tcommand'" perl -e 'exec $ENV{MY_COMMAND} ? $ENV{MY_COMMAND} : ("echo", "-e", "foo", "\n")';
```